db/migrate/20220217104818_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Like do |t|
+      t.references :User, null: false, foriegn_key: true, index:true
```

Comment:
- [ ] <b>REQUIRED:</b> Remember for User, the project requires to use `author_id`. You can check it out in the given [ERD Diagram.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) so we do it like this
```rb
t.references :author, foreign_key: { to_table: 'users' }